nautobot/core/templates/generic/object_bulk_import.html

```diff
@@ -33,84 +34,24 @@ <h1>{% block title %}{{ obj_type|bettertitle }} Bulk Import{% endblock %}</h1>
                         </div>
                     </form>
                     <div class="clearfix"></div>
-                    <p></p>
-                    {% if fields %}
-                        <div class="panel panel-default">
-                            <div class="panel-heading">
-                                <strong>CSV Field Options</strong>
+                    {% include 'generic/inc/bulk_import_field_options.html' %}
+                </div>
+                <div role="tabpanel" class="tab-pane {% if form.csv_file.errors %}active{% endif %}" id="csv-file">
```

Comment:
It'd be nice from a UX perspective if the two tab-panes were the same height, so that the "CSV Field Options" table doesn't jump up and down on the page when toggling between the tabs. Not sure how trivial that is to accomplish in Bootstrap 3 CSS though.